feature-extensions/apigee/terraform/variables.tf

```diff
@@ -0,0 +1,4 @@
+# These variables are set by CLI args from the calling script
+variable "region" { type = "string" default = "us-east-1" }
+variable "jazz_aws_accountid" {type = "string"}
+variable "env_prefix" {type = "string"}
```

Comment:
Perhaps call this `stack_prefix` here and other places?

Reply:
was attempting to match the existing installer variable name, but agree.